package.json

```diff
@@ -22,7 +22,7 @@
     "affected:dep-graph": "nx affected:dep-graph",
     "affected": "nx affected",
     "contributors:add": "all-contributors add",
-    "format": "nx format:write && yarn prettier --config .prettierrc --write \"**/*.*\" \"!dist/**\" \"!yarn.lock\"",
+    "format": "nx format:write && pnpx prettier --config .prettierrc --write \"**/*.*\" \"!dist/**\" \"!pnmpm-lock.yaml\"",
```

Comment:
I had this question before, why use **pnpx** having prettier as a **devDependency**?

Reply:
PNPM is kind of a mix of NPM and Yarn. It will run scripts like Yarn, but the `pnpm` executable won't run local packages. It has `pnpx` for that similar to NPM's `npx`.